_topic_map.yml

```diff
@@ -242,10 +242,12 @@ Topics:
   Dir: installing_vsphere
   Topics:
   - Name: Installing a cluster on vSphere
+    File: installing-vsphere-installer-provisioned
+  - Name: Installing a cluster on vSphere with user-provisioned infrastructure
     File: installing-vsphere
-  - Name: Installing a cluster on vSphere with network customizations
+  - Name: Installing a cluster on vSphere with user-provisioned infrastructure and network customizations
     File: installing-vsphere-network-customizations
-  - Name: Restricted network vSphere installation
+  - Name: Restricted network vSphere installation with user-provisioned infrastructure
```

Comment:
vSphere IPI also support installing a cluster under restricted network, this part is missed here.

Reply:
@patrickdillon, @jinyunma, do you have any information about the differences in the restricted network IPI installation?

Reply:
About restricted network IPI installation, except adding `imageContentSources` and `additionalTrustBundle`(desdribed irestriction netwrok UPI installation) into install-config.yaml, below parameter also need to be added into install-config.yaml before deploying cluster(openshift-install create cluster):
`clusterOSImage` (https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L1363)
And user must prepare one http server as per-conditoin,  download rhcos image and upload it to http server, then provide the file's http url to parameter clusterOSImage.
The location of this parameter in install-config.yaml is `platform.vsphere.clusterOSImage`

Reply:
Thank you! I'd like to get this PR merged ASAP, so I'm taking a follow-up issue to add the restricted networks use case: https://issues.redhat.com/browse/OSDOCS-1168